### PR TITLE
Add google analytics tag manager to application.html.erb view

### PR DESCRIPTION
**Pull request by djbroccoli**
Hey Santiago,

I don't know if this was the right way to do this...let me know. 

-Brock

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,7 +4,13 @@
...
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-WKCQDTL');</script>
+    <!-- End Google Tag Manager -->
```
Comment (san3091) [inline app/views/layouts/application.html.erb:13]: I think we should swap the tags, have the noscript in the head and this setup script in the body after the `yield`

**Review by san3091 — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (djbroccoli) [reply app/views/layouts/application.html.erb:13]: Word, I can make that change. This is the email from Sam I was following: 
"TAG MANAGER

For every page of website:
Past this code as high in the <head> of the page as possible


<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WKCQDTL');</script>


Paste this code immediately after the opening <body> tag.


<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WKCQDTL"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
 "

Comment (san3091) [reply app/views/layouts/application.html.erb]: oh shoot, in that case nevermind! google knows best, you can revert the change and I'll approve so you can merge.

**Review by san3091 — APPROVED** [review]
revert it back to google's holy instructions and then go ahead and merge!

**Pull request merged**